dashing/distribution.yaml

```diff
@@ -533,6 +533,21 @@ repositories:
       url: https://github.com/ros2/console_bridge_vendor.git
       version: dashing
     status: maintained
+  contracts_lite_vendor:
```

Comment:
Why add a "_vendor" suffix? It looks like the released CMake package is simply called `contracts_lite`.

Reply:
@jacobperron `contracts_lite` started life as a module in the Autoware.Auto project; that suffix is a convention for these kinds of wrapper packages.